src/com/amazon/ion/impl/bin/IonRawBinaryWriter.java

```diff
@@ -420,6 +427,10 @@ public void setFieldNameSymbol(final SymbolToken name)
         setFieldNameSymbol(name.getSid());
     }
 
+    public WriteBuffer getCurrentBuffer() {
```

Comment:
It looks like this can have default visibility, instead of public.

Reply:
> It looks like this can have default visibility, instead of public.

Thanks for catching this, will set to default in the next commit.